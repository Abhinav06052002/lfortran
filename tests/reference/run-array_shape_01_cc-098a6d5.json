{
    "basename": "run-array_shape_01_cc-098a6d5",
    "cmd": "lfortran --continue-compilation --no-color {infile}",
    "infile": "tests/errors/array_shape_01_cc.f90",
    "infile_hash": "2b1137cffda510c61749540ff4ba4792a3ca8b03d8f40d698c9154ce",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "run-array_shape_01_cc-098a6d5.stdout",
    "stdout_hash": "a779ee4f7341ddce58a41cee3d249ecf1b37216d7d4f16eb4e0f528e",
    "stderr": "run-array_shape_01_cc-098a6d5.stderr",
    "stderr_hash": "f0ad68882c983a64f69aa9d32d881deaf88620b5cf9b7b1d094f1adc",
    "returncode": 1
}