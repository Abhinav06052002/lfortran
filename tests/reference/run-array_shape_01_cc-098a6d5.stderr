semantic error: Different shape for array assignment on dimension 1(3 and 2)
 --> tests/errors/array_shape_01_cc.f90:3:1
  |
3 | x = reshape([1,2,3,4],[2,2])
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: Different shape for array assignment on dimension 1(3 and 1)
 --> tests/errors/array_shape_01_cc.f90:4:1
  |
4 | x = reshape([1,2,3,4],[1,2])
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
